keyboards/sol/keymaps/brianweyer/config.h

```diff
@@ -20,8 +20,6 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #pragma once
 
-#define SSD1306OLED
```

Comment:
Can you remove the user keymap changes?

Reply:
No, because they wanted me update it. I'll get them in here to approve.